python/ray/data/_internal/logical/rules/predicate_pushdown.py

```diff
@@ -111,32 +118,106 @@ def _try_push_down_predicate(cls, op: LogicalOperator) -> LogicalOperator:
             # Push the predicate down and return the result without the filter
             return input_op.apply_predicate(predicate_expr)
 
+        # Case 2: Check if operator allows predicates to pass through
+        if isinstance(input_op, PredicatePassThrough):
```

Comment:
Nit: `LogicalOperatorSupportsPredicatePushdown` and `PredicatePassThrough` appear to follow very different naming convention. Any reason not to use similar naming patterns (e.g., `PredicatePushdown`/`PredicatePassThrough` or `LogicalOperatorSupportsPredicatePushdown`/`LogicalOperatorSupportsPredicatePassThrough`)?